### PR TITLE
Prepare 0.68 release

### PR DESCRIPTION
**Pull request by ArthurSens**
## Description

Add Changelog entry and update generated files

### CODE REVIEW

**Comment by slashpai** [comment]
https://github.com/prometheus-operator/prometheus-operator/pull/5882 could go in this release too I guess? :)

**Comment by ArthurSens** [comment]
> #5882 could go in this release too I guess? :)

I've made one single comment there, but it is pretty small. I'm not against waiting for that to cut the release :)

**Comment by slashpai** [comment]
> > #5882 could go in this release too I guess? :)
> 
> I've made one single comment there, but it is pretty small. I'm not against waiting for that to cut the release :)

addressed :)

**Comment by slashpai** [comment]
Just have created https://github.com/prometheus-operator/prometheus-operator/pull/5897  was waiting for Prometheus `v2.47`, would be great to have this too. Looking for review :)

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,16 @@
+## 0.68.0 / 2023-09-06
+
+* [FEATURE] Add support for Webex receiver to the AlertmanagerConfig CRD. #5305
+* [FEATURE] Add support for MetricRelabelings to the ScrapeConfig CRD. #5805
+* [FEATURE] Add `dns_sd_configs` fields to the ScrapeConfig CRD. #5866
```
Comment (simonpasquier) [inline CHANGELOG.md:5]: (nit)
```suggestion
* [FEATURE] Add support for DNS service discovery to the ScrapeConfig CRD. #5866
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+* [FEATURE] Don't trigger reconcile loops if statefulsets have different `revisionHistoryLimit`. #5773
+* [ENHANCEMENT] Decreased CPU usage when reconciling ThanosRuler resources. #5784
```
Comment (simonpasquier) [inline CHANGELOG.md:7]:
```suggestion
* [ENHANCEMENT] Decrease CPU usage when reconciling ThanosRuler resources. #5784
```

**Comment by simonpasquier** [comment]
#5891 should be ready to go and a useful fix :)

**Comment by ArthurSens** [comment]
Comments addressed and also added entries for the latest PRs. (still waiting for the keep_dropped_targets PR)

**Comment by slashpai** [comment]
> Comments addressed and also added entries for the latest PRs. (still waiting for the keep_dropped_targets PR)

I updated the pr with changes, another review please :)

**Review by simonpasquier — APPROVED** [review]
:shipit:

**Pull request merged**